k-distribution/src/main/scripts/lib/pyk/kast.py

```diff
@@ -65,26 +65,86 @@ def KToken(token, sort):
 def isKToken(k):
     return k["node"] == "KToken"
 
+def KRewrite(lhs, rhs, att = None):
+    return { "node": "KRewrite", "lhs": lhs, "rhs": rhs, "att": att }
+
+def isKRewrite(k):
+    return k["node"] == "KRewrite"
+
 def KAtt(key, value):
     return {"node": "KAtt", "key": key, "value": value}
 
 def isKAtt(k):
     return k["node"] == "KAtt"
 
-def KRule(rule, requires = None, ensures = None, atts = None):
-    return { "node": "KRule", "rule": rule, "requires": requires, "ensures": ensures, "atts": atts }
+def KRule(rule, requires = None, ensures = None, att = None):
+    return { "node": "KRule", "body": rule, "requires": requires, "ensures": ensures, "att": att }
 
 def isKRule(k):
     return k["node"] == "KRule"
 
-def KImport(kimport):
-    return { "node": "KImport", "import": kimport }
+def KBubble(sentenceType, contents, att = None):
+    return { "node": "KBubble", "sentenceType": sentenceType, "contents": contents, "att": att }
+
+def isKBubble(k):
+    return k['node'] == 'KBubble'
+
+def KModuleComment(comment, att = None):
+    return { "node": "KModuleComment", "commonte": comment, "att": att }
```

Comment:
Is "commonte" the intended spelling here?

Reply:
fixed